binancial/data/get_trades_historical.py

```
@@ -1,4 +1,8 @@
-def get_trades_historical(client, symbol="BTCUSDT"):
+from typing import Any
+import pandas as pd
+
+def get_trades_historical(client: Any, symbol: str = "BTCUSDT", limit: int = 1000) -> pd.DataFrame:
+    ...
```

Ruff (0.8.2)

Unnecessary `...` literal

Remove unnecessary `...`

(PIE790)